docs/moderator-handbook.md

```diff
@@ -502,12 +503,12 @@ If the forum members determine there is nothing wrong with your code, you can re
 Thank you and happy coding.
 ```
 
-> When an issue is duplicate of an earlier issue
+> When an issue is duplicate of an earlier issue.
 
 ```markdown
 Thank you for reporting this issue.
 
-This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so we are closing it as a duplicate.
+This is a standard message notifying you that this issue appears to be very similar to issue <#number>, so we are closing it as a duplicate.
```

Comment:
```suggestion
This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so we are closing it as a duplicate.
```

as this is a prewritten message, having it as `#XXXXX` means we have to change less characters, as that's the format used on GitHub to reference issues and PRs, like `#47039` becoming #47039